relayer/crates/starknet-chain-components/src/impls/payload_builders/create_client.rs

```diff
@@ -62,3 +65,22 @@ where
         })
     }
 }
+
+pub struct ProvideNoCreateClientMessageOptionsOverride;
+
+#[cgp_provider(OverrideCreateClientPayloadOptionsComponent)]
+impl<Chain, Counterparty> ProvideOverrideCreateClientPayloadOptions<Chain, Counterparty>
+    for ProvideNoCreateClientMessageOptionsOverride
+where
+    Chain: HasCreateClientPayloadOptionsType<
+        Counterparty,
+        CreateClientPayloadOptions = StarknetCreateClientPayloadOptions,
+    >,
+{
+    fn override_create_client_payload_options(
+        payload_options: &StarknetCreateClientPayloadOptions,
+        _new_period: Duration,
+    ) -> StarknetCreateClientPayloadOptions {
+        payload_options.clone()
+    }
+}
```

Comment:
why has the payload not been updated here ? isn't it for tendermint client on starknet ?

Reply:
This is for:
```
pub struct StarknetCreateClientPayloadOptions {
    pub wasm_code_hash: [u8; 32],
}
```
So I don't see any utility to updating it